.github/workflows/tests.yml

```diff
@@ -37,41 +37,43 @@ jobs:
         if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
         run: poetry install --no-interaction --no-root
 
-  setup_cache_monetdb:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check out repository
-        uses: actions/checkout@v2
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-
-      - name: Load MONETDB cached image
-        uses: actions/cache@v2
-        with:
-          path: /tmp/.buildx-cache/monetdb
-          key: ${{ runner.os }}-buildx-monetdb-${{hashFiles( 'monetdb/**' )}}
-          restore-keys: |
-            ${{ runner.os }}-buildx-monetdb-
-
-      - name: Build MONETDB docker image
-        uses: docker/build-push-action@v2
-        with:
-          context: .
-          file: monetdb/Dockerfile
-          push: false
-          load: true
-          tags: madgik/mipenginedb:latest
-          cache-from: type=local,src=/tmp/.buildx-cache/monetdb
-          cache-to: type=local,dest=/tmp/.buildx-cache-new/monetdb
-
-        # Temp fix
-        # https://github.com/docker/build-push-action/issues/252
-        # https://github.com/moby/buildkit/issues/1896
-      - name: Move Docker images cache
-        run: |
-          rm -rf /tmp/.buildx-cache
-          mv /tmp/.buildx-cache-new /tmp/.buildx-cache
+#  Caching monetdb doesn't work because "apt-get update" in the Dockerfile messes up the caching,
```

Comment:
can you explain a bit more what you mean by that?

Reply:
I will make a ticket for it but I dont think it makes sense to actually fix this. Its not an actual problem its a caching problem for the automated tests. I will leave the comment as is and make a ticket to discuss it later and decide.

Reply:
I will make a ticket for it but I dont think it makes sense to actually fix this. Its not an actual problem its a caching problem for the automated tests. I will leave the comment as is and make a ticket to discuss it later and decide.